modules/bibcirculation/lib/bibcirculation_templates.py

```diff
@@ -1970,6 +1971,149 @@ def tmpl_borrower_search_result(self, result, redirect_to_new_request=False,
+        <br />
+        <br />
+        <br />
+        <input type=hidden value="0">
+        <input type=hidden value="10">
+        <table class="bibcirctable">
+          <tr align="center">
+            <td class="bibcirctableheader">Calculate Loan Period
+              <br />
+              <br />
+            </td>
+          </tr>
+          <tr align="center">
+          <td> <b>Start Date :</b>(Format : 2014-01-01 00:00:00) <input type="text" size="20" name="s" style='border: 1px solid #cfcfcf'> </td>
+                             </tr>
+
+        <tr align="center">
+          <td><b>End Date :</b>(Format : 2015-06-01 09:00:01) <input type="text" size="20" name="e" style='border: 1px solid #cfcfcf'></td> 
+                             </tr>
+        <tr align="center">
+          <td><b>Number of Top Results :</b>(Format : 10) <input type="text" size="20" name="n" style='border: 1px solid #cfcfcf'></td>
+                             </tr>
+
+        </table>
+        <br />
+        <table class="bibcirctable">
+          <tr align="center">
+            <td>
+
+              <input type=button value='%s'
+              onClick="history.go(-1)" class="formbutton">
+              <input type="submit" value='%s' class="formbutton">
+
+            </td>
+          </tr>
+        </table>
+        <br />
+        <br />
+        <br />
+        <br />
+        </div>
+        <form>
+
+        """ % (CFG_SITE_URL, _("Back"), _("Search"))
+
+        return out
+    
+    def tmpl_loan_period_result(self, result, ln=CFG_SITE_LANG):
+        """
+        @param ln: language of the page
+        """
+
+        _ = gettext_set_language(ln)
+
+        out = """  """
+
+        out += load_menu(ln)
+
+        if result == None:
+            out += """
+            <div class="bibcircbottom">
+            <br />
+            <div class="infoboxmsg">%s</div>
+            <br />
+            """ % (_("0 item(s) found."))
+
+        else:
+            out += """
+        <style type="text/css"> @import url("/img/tablesorter.css"); </style>
+        <div class="bibcircbottom">
+        <br />
+        <table class="bibcirctable">
+          <tr align="center">
+            <td>
+               <strong>%s item(s) found</strong>
+            </td>
+          </tr>
+        </table>
+        <br />
+          <table class="tablesorter" border="0" cellpadding="0" cellspacing="1">
+            <thead>
+              <tr>
+                <th>%s</th>
+                <th>%s</th>
+                <th>%s</th>
+                <th>%s</th>
+		<th>%s</th>
+              </tr>
+            </thead>
+          <tbody>
+        """ % (len(result), _("Recorrd No"), _("No of items"),
+	 _("Loan Period(in days)"), _("Item List"), 
+               _("Volume"))
+
+            for recid in result:
+
+                (volume, loanPeriod, itemList, noOfItems, recordNo) = recid       
```

Comment:
I think you can merge this two lines, i.e. `for (volume, loanPeriod, itemList, noOfItems, recordNo) in result`